aptos-move/block-executor/src/view.rs

```diff
@@ -1177,12 +1177,14 @@ impl<'a, T: Transaction, S: TStateView<Key = T::Key>, X: Executable> LatestView<
         delayed_write_set_keys: &HashSet<T::Identifier>,
         skip: &HashSet<T::Key>,
     ) -> Result<BTreeMap<T::Key, (T::Value, u64)>, PanicError> {
-        parallel_state
+        let reads_with_delayed_fields = parallel_state
             .captured_reads
             .borrow()
             .get_group_read_values_with_delayed_fields(skip)
             .map(|(k, v)| (k.clone(), v.clone()))
```

Comment:
nit: cloned()?

Reply:
actually, that doesn't work - we don't have a reference to pair, but pair of references. so cloned() cannot be applied there, and we need to clone the elements themselves

Reply:
I see, good to know, thanks!